src/ui/js/Overview.js

```diff
@@ -369,6 +378,94 @@ Overview.pageAddGameTable = function(
   }
 };
 
+Overview.pageAddGameTableNew = function() {
+  var gamesource = Api.new_games.games;
+  var tableClass = 'newGames';
+
+  if (gamesource.length === 0) {
+    return;
+  }
+
+  var tableBody = Overview.page.find('table.' + tableClass + ' tbody');
+  var tableDiv = $('<div>');
+  tableDiv.append($('<h2>', {'text': 'New games', }));
+  var table = $('<table>', { 'class': 'gameList ' + tableClass, });
+  tableDiv.append(table);
+  Overview.page.append(tableDiv);
+
+  var tableHead = $('<thead>');
+  var headerRow = $('<tr>');
+  headerRow.append($('<th>', {'text': 'Game #', }));
+  headerRow.append($('<th>', {'text': 'Your Button', }));
+  headerRow.append($('<th>', {'text': 'Opponent\'s Button', }));
+  headerRow.append($('<th>', {'text': 'Opponent', }));
+  headerRow.append($('<th>', {'text': 'Max wins', }));
+  headerRow.append($('<th>', {'text': 'Action', 'colspan': '2', }));
+
+  tableHead.append(headerRow);
+  table.append(tableHead);
+
+  tableBody = $('<tbody>');
+  table.append(tableBody);
+
+  var i = 0;
+  var gameInfo;
+  var gameRow;
+  var decideTd;
+  var acceptLink;
+  var rejectLink;
+  while (i < gamesource.length) {
+    gameInfo = gamesource[i];
+
+    gameRow = $('<tr>');
+    gameRow.append($('<td>', {
+      'text': 'Game ' + gameInfo.gameId,
+    }));
+    gameRow.append($('<td>').append(
+      Env.buildButtonLink(gameInfo.playerButtonName)
+    ));
+    gameRow.append($('<td>').append(
+      Env.buildButtonLink(gameInfo.opponentButtonName)
+    ));
+    gameRow.append($('<td>').append(
+      Env.buildProfileLink(gameInfo.opponentName)
+    ));
+    gameRow.append($('<td>', {
+      'text': gameInfo.maxWins,
+    }));
+
+    decideTd = $('<td>');
+    gameRow.append(decideTd);
+
+    if (Api.new_games.games[0].isAwaitingAction) {
+      acceptLink = $('<a>', {
+        'text': 'Accept',
+        'href': '#',
+        'data-gameId': gameInfo.gameId,
+      });
+      acceptLink.click(Overview.formAcceptGame);
+
+      decideTd.append('[')
+              .append(acceptLink)
+              .append('] / ');
+    }
+
+    rejectLink = $('<a>', {
+      'text': 'Reject',
+      'href': '#',
+      'data-gameId': gameInfo.gameId,
```

Comment:
Right, I've found where I copied it from, namely the Dismiss Game code.

If you do a global search, though, on "data-", you'll find things like "data-boardId", "data-postId", "data-column", "data-direction" throughout all the API code.